WelcomeHome/WelcomeHome.DAL/Repositories/EventsRepository.cs

```diff
@@ -0,0 +1,53 @@
+﻿using WelcomeHome.DAL.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace WelcomeHome.DAL.Repositories
+{
+    public class EventsRepository : IEventsRepository
+    {
+        private WelcomeHomeDbContext _context;
+
+        public EventsRepository(WelcomeHomeDbContext context)
+        {
+            this._context = context;
+        }
+
+        public async Task<IEnumerable<Event>> GetEventsAsync()
+        {
+            return await _context.Events.ToListAsync().ConfigureAwait(false);
```

Comment:
Do not use toList, better use Select. We will need to retrieve all data in a final pont (controller or service)